frontstage/views/surveys/upload_survey_failed.py

```diff
@@ -14,7 +15,17 @@
 @jwt_authorization(request)
 def upload_failed(session):
     case_id = request.args.get('case_id', None)
+    party_id = request.args.get('party_id', None)
```

Comment:
Can get party_id from the session instead of passing it in. party_id = session['party_id']